src/main/java/cz/muni/fi/pv286/mmt/model/Options.java

```diff
@@ -0,0 +1,163 @@
+
+    /**
+     * Gets the input format.
+     */
+    public IoFormat getInputFormat() {
+        return inputFormat;
+    }
+
+    /**
+     * Sets the input format.
+     */
+    public void setInputFormat(IoFormat inputFormat) {
+        if (inputFormat == IoFormat.INT && !wasFromOptionSet) {
+            inputFromToOption = Optional.of(IoOption.BIG);
+        }
+        if (inputFormat == IoFormat.BITS && !wasFromOptionSet) {
+            inputFromToOption = Optional.of(IoOption.LEFT);
+        }
+        this.inputFormat = inputFormat;
+    }
+
+    /**
+     * Get the input FromToOption.
+     */
+    public Optional<IoOption> getInputFromToOption() {
+        return inputFromToOption;
+    }
+
+    /**
+     * Sets the input FromToOption.
+     */
+    public void setInputFromToOption(FromToOption inputFromToOption) {
+        this.inputFromToOption = Optional.of(IoOption.from(inputFromToOption));
+        wasFromOptionSet = true;
+    }
+
+
+    /**
+     * Sets the bracket type option.
+     */
+    public void setBracketOption(BracketType bracketType) {
+        this.bracketType = Optional.of(bracketType);
+        wasBracketSet = true;
+    }
+
+    /**
+     * Gets the output format.
+     */
+    public IoFormat getOutputFormat() {
+        return outputFormat;
+    }
+
+    /**
+     * Sets the output format.
+     */
+    public void setOutputFormat(IoFormat outputFormat) {
+        if (outputFormat == IoFormat.INT && !wasToOptionSet) {
+            outputFromToOption = Optional.of(IoOption.BIG);
+        }
+        if (outputFormat == IoFormat.ARRAY && !wasToOptionSet) {
+            outputFromToOption = Optional.of(IoOption.HEX);
+        }
+        if (outputFormat == IoFormat.ARRAY && !wasBracketSet) {
+            bracketType = Optional.of(BracketType.CURLY_BRACKET);
+        }
+
+        this.outputFormat = outputFormat;
+    }
+
+    /**
+     * Gets the output FromToOption.
+     */
+    public Optional<IoOption> getOutputFromToOption() {
+        return outputFromToOption;
+    }
+
+    /**
+     * Sets the output FromToOption.
+     */
+    public void setOutputFromToOption(FromToOption outputFromToOption) {
+        if (outputFromToOption == FromToOption.CURLY_BRACKET) {
+            setBracketOption(BracketType.CURLY_BRACKET);
+        } else if (outputFromToOption == FromToOption.SQUARE_BRACKET) {
+            setBracketOption(BracketType.SQUARE_BRACKET);
+        } else if (outputFromToOption == FromToOption.REGULAR_BRACKET) {
+            setBracketOption(BracketType.REGULAR_BRACKET);
+        } else {
+            this.outputFromToOption = Optional.of(IoOption.from(outputFromToOption));
+            wasToOptionSet = true;
+        }
+    }
+
+    public InputStream getInputFile() {
+        return inputFile;
+    }
+
+    public void setInputFile(InputStream inputFile) {
+        this.inputFile = inputFile;
+    }
+
+    public OutputStream getOutputFile() {
+        return outputFile;
+    }
```

Comment:
**issue:** Anyone can modify contents of output stream.

Since this method is public anyone is able to write anything in this output stream.